capture/parsers/isis.c

```diff
@@ -78,4 +119,11 @@ void moloch_parser_init()
                                              isis_pre_process,
                                              isis_process,
                                              NULL);
+
+
+    typeField = moloch_field_define("isis","uptermfield",
```

Comment:
looks like this should be lotermfield since everything is lowercase

Reply:
k made these changes